homeassistant/components/tesla/device_tracker.py

```diff
@@ -1,19 +1,20 @@
 """Support for tracking Tesla cars."""
 import logging
 
-from homeassistant.helpers.event import track_utc_time_change
+from homeassistant.helpers.event import async_track_utc_time_change
 from homeassistant.util import slugify
 
 from . import DOMAIN as TESLA_DOMAIN
 
 _LOGGER = logging.getLogger(__name__)
 
 
-def setup_scanner(hass, config, see, discovery_info=None):
+async def async_setup_scanner(hass, config, async_see, discovery_info=None):
     """Set up the Tesla tracker."""
-    TeslaDeviceTracker(
-        hass, config, see, hass.data[TESLA_DOMAIN]["devices"]["devices_tracker"]
+    tracker = TeslaDeviceTracker(
+        hass, config, async_see, hass.data[TESLA_DOMAIN]["devices"]["devices_tracker"]
     )
+    async_track_utc_time_change(hass, tracker.update_info, second=range(0, 60, 30))
```

Comment:
Should we call `tracker.update_info` here, besides scheduling it, to retain the old behavior?

Reply:
It probably doesn't matter because the config_entry version doesn't use this tracker anyway. Also the underlying dependency is just feeding a cached copy of the location data anyway which is updated on it's own cycle independent of the device tracker.

Reply:
This update would be to provide the initial state sooner than the update interval.

Reply:
Ok.